client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -0,0 +1,135 @@
+	};
+
+	toggleExpanded = () => {
+		this.setState( { isExpanded: ! this.state.isExpanded } );
+	}
+
+	render() {
+		const {
+			authorAvatarUrl,
+			authorDisplayName,
+			authorEmail,
+			authorIp,
+			authorIsBlocked,
+			authorUrl,
+			authorUsername,
+			blockUser,
+			commentDate,
+			moment,
+			translate,
+		} = this.props;
+		const { isExpanded } = this.state;
+
+		const classes = classNames( 'comment-detail__author', {
+			'is-expanded': isExpanded,
+		} );
+
+		return (
+			<div className={ classes }>
+				<div className="comment-detail__author-preview">
+					<div className="comment-detail__author-avatar">
+						<img
+							className="comment-detail__author-avatar-image"
+							src={ authorAvatarUrl }
+						/>
+					</div>
+					<div className="comment-detail__author-info">
+						<div className="comment-detail__author-info-element comment-detail__author-name">
+							<strong>
+								{ authorDisplayName }
+							</strong>
+							<span>
+								{ authorUrl }
+							</span>
+						</div>
+						<div className="comment-detail__author-info-element comment-detail__comment-date">
+							{ moment( commentDate ).format( 'MMMM D, YYYY H:mma' ) }
+						</div>
+					</div>
+					<a className="comment-detail__author-more-info-toggle" onClick={ this.toggleExpanded }>
+						<Gridicon icon="info-outline" />
+					</a>
+				</div>
+
+				<div className="comment-detail__author-more-info">
+					<div className="comment-detail__author-more-actions">
+						<div className="comment-detail__author-more-element comment-detail__author-more-element-author">
+							<div className="comment-detail__author-avatar">
+								<img className="comment-detail__author-avatar-image" src={ authorAvatarUrl } />
+							</div>
+							<div className="comment-detail__author-info">
+								<div className="comment-detail__author-name">
+									<strong>
+										{ authorDisplayName }
+									</strong>
+								</div>
+								<div className="comment-detail__author-username">
+									{ authorUsername }
+								</div>
+							</div>
+						</div>
+						<div className="comment-detail__author-more-element">
+							<Gridicon icon="mail" />
+							<span>
+								{ authorEmail }
+							</span>
+						</div>
+						<div className="comment-detail__author-more-element">
+							<Gridicon icon="link" />
+							<span>
+								{ authorUrl }
+							</span>
+						</div>
+						<div className="comment-detail__author-more-element">
+							<Gridicon icon="globe" />
+							<span>
+								{ authorIp }
+							</span>
+						</div>
+					</div>
+					<div className="comment-detail__author-more-actions">
+						<a
+							className={ classNames(
+								'comment-detail__author-more-element comment-detail__author-more-element-block-user',
+								{ 'is-blocked': authorIsBlocked }
+							) }
+							onClick={ blockUser }
+						>
+							<Gridicon icon="block" />
+							<span>{
+								authorIsBlocked
+									? translate( 'Unblock user' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Unblock' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).